src/multipatch/connectivity/edge_transformation.hpp

```diff
@@ -11,6 +11,8 @@
 #include "edge.hpp"
 #include "interface.hpp"
 
+template <class TargetGrid>
+static constexpr Idx<TargetGrid> out_of_bounds_idx = Idx<TargetGrid>(std::string::npos);
```

Comment:
```suggestion
template <class TargetGrid>
static constexpr Idx<TargetGrid> out_of_bounds_idx = Idx<TargetGrid>(Kokkos::finite_max_v<ddc::DiscreteElementType>);
```

Just a minor remark, it will not work in a device function that takes a reference to this global variable.

Reply:
I don't know but I wouldn't be surprised if we did

Reply:
> What won't be a template variable? out_of_bounds_idx?

Yes

> How would you do this as a template function? Why would it likely not be a template variable?

You can do something like:
```cpp
template <class TargetGrid>
constexpr Idx<TargetGrid> out_of_bounds_idx()
{
    return Idx<TargetGrid>(std::numeric_limits<ddc::DiscreteElementType>::max());
}
```

> Why do I need to care about the final DDC implementation for this PR?

As you wish ^^'